File: BOJ/11399/A.py
import sys

sys.stdin = open("D:\Projects\AA\BOJ\\11399\input.txt", "r")
input = sys.stdin.readline

N = int(input().rstrip())
arr = []
arr = list(map(int, input().rstrip().split()))
arr.sort()
sum = arr[0]
ans = sum
for i in range(1, len(arr)):
    ans = ans + sum + arr[i]
    sum = sum + arr[i]
print(ans)

File: BOJ/11399/A.cpp
#include <iostream>
#include <algorithm>
#include <vector>
using namespace std;

int main()
{
    ios::sync_with_stdio(0);
    cin.tie(0);
    freopen("input.txt", "rt", stdin);
    int N;
    cin >> N;
    vector<int> arr;
    for (int i = 0; i < N; i++)
    {
        int temp;
        cin >> temp;
        arr.push_back(temp);
    }
    sort(arr.begin(), arr.end());
    int ans = 0;
    for (int i = 0; i < N; i++)
    {
        ans = ans + arr[i] * (N - i);
    }
    cout << ans;
}